readline.js

```diff
@@ -18,9 +17,16 @@ const autoComplete = new AutoComplete({
 });
 
 async function run() {
+  try {
     const commandToRun = await select.run();
     const projectName = await autoComplete.run();
-    launchBackstop(commandToRun, projectName);
+    const availableLocalHost = await checkLocalHost(projectName);
+    if (availableLocalHost === true) {
```

Comment:
если ты в резовл передаешь булиан, тогда не нужно тут на тру сравнивать
```suggestion
    if (availableLocalHost) {
```